sdk/python/tests/test_feature_store.py

```diff
@@ -41,7 +41,7 @@ def feature_store_with_local_registry():
     fd, online_store_path = mkstemp()
     return FeatureStore(
         config=RepoConfig(
-            registry=registry_path,
+            registry=f"{registry_path}.pb",
```

Comment:
https://github.com/feast-dev/feast/pull/1653#discussion_r659024618 We are the folks creating files without an extension 😄 
![image](https://user-images.githubusercontent.com/31944154/123493134-70f5c180-d5d0-11eb-9d75-a18eae1495a2.png)

Reply:
lololol